third_party/ruby_externals.bzl

```diff
@@ -99,3 +128,12 @@ def _ruby_urls(path):
         "https://cache.ruby-lang.org/pub/ruby/{}".format(path),
         "https://artifactory-content.stripe.build/artifactory/ruby-lang-cache/pub/ruby/{}".format(path),
     ]
+
+def _github_public_urls(path):
```

Comment:
this is copied from `third_party/externals.bzl` - do you want me to make it public so we can share across files instead?